src/server/segcache/benches/benchmark.rs

```diff
@@ -36,7 +36,7 @@ fn get_benchmark(c: &mut Criterion) {
     let mut buffer = vec![0; 1024 * 1024];
 
     // define a benchmarking group
-    let mut group = c.benchmark_group("request");
+    let mut group = c.benchmark_group("request_seg");
```

Comment:
Why the change here?

Reply:
I may have misinterpreted a need to have the bench names be unique to also extend to benchmark groups, which probably isnt the case. I can change it back